ext/datadog_profiling_native_extension/ruby_helpers.c

```diff
@@ -11,43 +11,141 @@ static VALUE module_object_space = Qnil;
 static ID _id2ref_id = Qnil;
 static ID inspect_id = Qnil;
 static ID to_s_id = Qnil;
+static ID new_id = 0;
+static ID telemetry_message_id = Qnil;
+
+void ruby_helpers_init(VALUE profiling_module) {
```

Comment:
I think the `profiling_module` argument is a leftover of a previous version -- I don't see it being used? 👀